lzy/execution-env/src/main/java/ai/lzy/env/aux/CondaPackageRegistry.java

```diff
@@ -14,167 +19,258 @@ public class CondaPackageRegistry {
     private static final Logger LOG = LogManager.getLogger(CondaPackageRegistry.class);
     private static final String VERSION_REGEX = "(?=>=)|(?=<=)|(?=<)|(?=>)|==|=";
     private static final int SPLIT_LIMIT = 2;
+    private static final String DEFAULT_ENV_NAME = "base";
```

Comment:
why not `default`?

Reply:
Because default conda env is base)
Now we don't have to install an extra env when creating the base image